File: cpp/program/ProdukAudioVisual.cpp
#include "ProdukElektronik.cpp" 

class ProdukAudioVisual : public ProdukElektronik {
protected:
    double harga;
    std::string garansi;
    std::string tipe;

public:
    // Constructor
    ProdukAudioVisual(const std::string& idProduk, const std::string& namaProduk, const std::string& merk,
                      double harga, const std::string& garansi, const std::string& tipe)
        : ProdukElektronik(idProduk, namaProduk, merk), harga(harga), garansi(garansi), tipe(tipe) {}

    // Getters
    double getHarga() const { return harga; }
    std::string getGaransi() const { return garansi; }
    std::string getTipe() const { return tipe; }

    // Setters
    void setHarga(double harga) { this->harga = harga; }
    void setGaransi(const std::string& garansi) { this->garansi = garansi; }
    void setTipe(const std::string& tipe) { this->tipe = tipe; }
};

File: python/program/produk_audio_visual.py
from produk_elektronik import ProdukElektronik

class ProdukAudioVisual(ProdukElektronik):
    def __init__(self, id_produk, nama_produk, merk, harga, garansi, tipe):
        super().__init__(id_produk, nama_produk, merk)
        self.__harga = harga
        self.__garansi = garansi
        self.__tipe = tipe

    # Getter
    def get_harga(self):
        return self.__harga

    def get_garansi(self):
        return self.__garansi

    def get_tipe(self):
        return self.__tipe

    # Setter
    def set_harga(self, harga):
        self.__harga = harga

    def set_garansi(self, garansi):
        self.__garansi = garansi

    def set_tipe(self, tipe):
        self.__tipe = tipe